content/en/docs/release-notes/rn-armory-operator/armory-operator-v1-1-0.md

```diff
@@ -0,0 +1,30 @@
+---
+title: v1.1.0 Armory Operator
+toc_hide: true
+---
+
+## 08/11/2020 Release Notes
+
+## Known Issues
+No known issues.
+
+## Armory Operator
+
+* chore(release): upgrade to oss operator 1.1.0
+* fix(kustomize): Make kustomization.yml more compatible
```

Comment:
It doesn't sound like it belongs in the release notes. Was just to make the example `SpinnakerService` compatible with both kubectl and kustomize